static/src/javascripts-legacy/projects/commercial/modules/dfp/render-advert.js

```diff
@@ -156,6 +164,42 @@ define([
                 }) : Promise.resolve();
             }
 
+            function addFeedbackDropdownToggle() {
+                return isRendered ? fastdom.write(function () {
+                    if (!bonzo(advert.node).hasClass('js-toggle-ready')){
+                        new Toggles(advert.node).init();
+                    }
+                }) : Promise.resolve();
+            }
+
+            function applyFeedbackOnClickListeners(slotRenderEvent) {
+                var readyClass = 'js-onclick-ready';
+                return isRendered ? fastdom.write(function () {
+                    bonzo(qwery('.popup__item-problem--option')).each(function(el) {
+                        var option = bonzo(el);
+                        if (option.hasClass(readyClass)) return;
```

Comment:
You can get rid of that and just call `qwery('.popup__item-problem--option.js-onclick-ready')` above